### PR TITLE
changes type to sls-app and yaml file changes

### PR DESCRIPTION
**Pull request by swati-sh**
Requirements
type to sls-app

### CODE REVIEW

File: core/jazz_ui/src/app/pages/environment-overview/env-overview-section.component.ts [hunk 1]
```diff
@@ -33,9 +33,10 @@ export class EnvOverviewSectionComponent implements OnInit {
   private env:any;
   branchname: any;
   friendlyChanged:boolean = false;
-  textChanged:boolean=false;
+  textChanged:boolean = true;
   tempFriendlyName:string;
   yaml:string = "";
+  textarea:boolean= true;
```
Comment (raghits) [inline core/jazz_ui/src/app/pages/environment-overview/env-overview-section.component.ts]: space around the = sign (across the board).

File: core/jazz_ui/src/app/pages/environment-overview/env-overview-section.component.ts [hunk 2]
```diff
@@ -249,15 +264,13 @@ popup(state){
...
-    this.showCancel=false;
-    this.saveButton=false;
-    this.editButton=true;
-    this.isDiv=true;
-    this.showCncl=false;
-    this.saveBtn=false;
-    this.editBtn=true;
-    this.yaml='';
-    this.isEditClicked=false;
+    this.showCncl = false;
```
Comment (raghits) [inline core/jazz_ui/src/app/pages/environment-overview/env-overview-section.component.ts]: showCncl different from showCancel? do we have both?

**Comment by raghits** [comment]
will go ahead and merge based on approval.

File: core/jazz_ui/src/app/pages/environment-overview/env-overview-section.component.ts (hunk 2 above)
Comment (swati-sh) [reply core/jazz_ui/src/app/pages/environment-overview/env-overview-section.component.ts]: yes

Comment (swati-sh) [reply core/jazz_ui/src/app/pages/environment-overview/env-overview-section.component.ts]: Two edit button is available on that page.

**Pull request merged**